nailgun/entities.py

```diff
@@ -1096,6 +1135,23 @@ def __init__(self, server_config=None, **kwargs):
         }
         super(ContentView, self).__init__(server_config, **kwargs)
 
+    def read(self, entity=None, attrs=None, ignore=()):
+        """Fetch an attribute missing from the server's response.
+
+        For more information, see `Bugzilla #1237257
+        <https://bugzilla.redhat.com/show_bug.cgi?id=1237257>`_.
+
+        """
+        if (getattr(self._server_config, 'version', parse_version('6.1')) <
```

Comment:
And we need that comparison to apply some workaround, right? Did we have different API in version before 6.1?